test/deploy-kubeflow.sh

```diff
@@ -56,4 +56,11 @@ ${KUBEFLOW_SRC}/scripts/kfctl.sh apply platform
 ${KUBEFLOW_SRC}/scripts/kfctl.sh generate k8s
 ${KUBEFLOW_SRC}/scripts/kfctl.sh apply k8s
 
+pushd ks_app
```

Comment:
do you think we can bump the version in kfctl instead?
https://github.com/kubeflow/kubeflow/blob/master/kubeflow/argo/prototypes/argo.jsonnet

Reply:
Sure, I saw kubeflow/kubeflow#3335 so I pushed the removal of these lines, along with the corresponding change in `test/install-argo.sh`.

Let's merge this upon the aforementioned PR's approval. Would that be ok?